actionview/lib/action_view/helpers/tags/base.rb

```diff
@@ -138,7 +138,7 @@ def sanitized_method_name
           end
 
           def sanitized_value(value)
-            value.to_s.gsub(/\s/, "_").gsub(/[^-\w]/, "").downcase
+            value.to_s.gsub(/\s/, "_").gsub(/[^-\p{Word}]/, "").downcase
```

Comment:
Without commenting on whether the change is good which I don't have the context to determine, could you use `[[:word:]]` instead of `\p{Word}`?  There former style has several uses in the codebase, the later has none.  Sticking to one style will help keep the codebase more approachable.